xapi-profiles-communication.md

```diff
@@ -37,60 +37,62 @@ Part Three:	[xAPI Profiles Communication and Processing Specification](./xapi-pr
 <a name="partthree"></a>
 # Part Three: Communication and Processing
 
-In addition to the ability to host Profiles as documents, there will be infrastructure 
-for querying and manipulating Profiles. This document describes the algorithms for 
-processing Profiles, including the exact rules by which Statement Templates and Patterns 
-are validated against Statements. It also describes a “Profile Server” to make it easier 
-to manage and answer questions about Profiles from a centralized location, including 
+In addition to the ability to host Profiles as documents, there will be infrastructure
+for querying and manipulating Profiles. This document describes the algorithms for
+processing Profiles, including the exact rules by which Statement Templates and Patterns
+are validated against Statements. It also describes a “Profile Server” to make it easier
+to manage and answer questions about Profiles from a centralized location, including
 implementing the algorithms.
 
 ## <a name="1.0">1.0</a> Profile Server
 
-A Profile Server manages xAPI Profiles from a centralized location. [An RDF triple store](https://en.wikipedia.org/wiki/Triplestore) is responsible for the storage of Profiles. 
+A Profile Server manages xAPI Profiles from a centralized location. [An RDF triple store](https://en.wikipedia.org/wiki/Triplestore) is responsible for the storage of Profiles.
 A Profile Server will allow administrators to add Profiles by their contents or by URI to the Profile Server
 
-A Profile Server will host a SPARQL endpoint containing the RDF information from the 
-contained Profiles at the path /sparql. This enables xAPI Profiles to be queried. SPARQL 
-servers have the ability to divide information into multiple datasets, or graphs, and 
-offer separate querying on them. One of these is the default graph, which is queried 
-when no other graph is specified. The default graph at this SPARQL endpoint will 
-contain all the current versions of Profiles. 
+A Profile Server will host a SPARQL endpoint containing the RDF information from the
+contained Profiles at the path /sparql. This enables xAPI Profiles to be queried. SPARQL
+servers have the ability to divide information into multiple datasets, or graphs, and
+offer separate querying on them. One of these is the default graph, which is queried
+when no other graph is specified. The default graph at this SPARQL endpoint will
+contain all the current versions of Profiles.
+
+A Profile Server will include inference logic for the following, at minimum: all SKOS predicate relationships, and `profile:concepts`, `profile:templates`, and `profile:patterns` being subproperties of the inverse of `skos:inScheme`.
```

Comment:
This is the essential paragraph. The other changes are just my editor removing trailing whitespace (and removing an extraneous sentence).